docs/items/overrides.md

```diff
@@ -0,0 +1,46 @@
+Item Property Overrides
+=======================
+
+Items do not have something analogous to blockstate JSONs, where you can define models in `models/block/` and then stitch them into one big model in a blockstate JSON under `blockstates/` depending on variants. Instead, you must assign an item a single model defined under `models/item/`. Then, the item will expose certain properties from the code. These properties are defined as a name string and a function that maps `ItemStack`s to arbitrary float values. The model JSON defines the default model for the item, and has an extra `"overrides"` tag, which is a list of predicates and a model to use if the predicate is true. The format can be seen on the [wiki][].
```

Comment:
It's just how it is. The definition of Markdown's syntax says that `[a][b]` creates a link with the text `a` that links to the URL specified by the reference `b`, and that `[a][]` is shorthand for `[a][a]`.